pkg/controllers/rollout/controller.go

```diff
@@ -333,82 +331,34 @@ func (r *Reconciler) pickBindingsToRoll(ctx context.Context, allBindings []*flee
 	readyTimeCutOff := time.Now().Add(-time.Duration(*crp.Spec.Strategy.RollingUpdate.UnavailablePeriodSeconds) * time.Second)
 
 	// classify the bindings into different categories
-	// TODO: calculate the time we need to wait for the first applied but not ready binding to be ready.
+	// Wait for the first applied but not ready binding to be ready.
 	// return wait time longer if the rollout is stuck on failed apply/available bindings
+	allReady := true
```

Comment:
is this still needed?

Reply:
yes, checks if they are all ready to be able to set the minimum waitTime to 0.